apps/portal/app/routes/app+/list+/$id+/index.tsx

```diff
@@ -246,146 +452,145 @@ export default function ListOverview() {
             <Suspense
```

Comment:
We can also add in the loading and error states like in this example from Data Created:

```
 <Suspense
            fallback={
              <div className="flex flex-col w-full gap-6">
                <DataHeaderSkeleton />
                <PaginatedListSkeleton />
              </div>
            }
          >
            {isLoadingAtomsCreated ? (
              <div className="flex flex-col w-full gap-6">
                <DataHeaderSkeleton />
                <PaginatedListSkeleton />
              </div>
            ) : isErrorAtomsCreated ? (
              <ErrorStateCard
                title="Failed to load identities created"
                message={
                  (errorAtomsCreated as Error)?.message ??
                  'An unexpected error occurred'
                }
              >
                <RevalidateButton />
              </ErrorStateCard>
            ) : (
              <TabContent
                value={DataCreatedHeaderVariants.createdIdentities}
                totalResults={atomsCreatedResult?.total?.aggregate?.count}
                atomImage={accountResult?.account?.image ?? ''}
                atomLabel={accountResult?.account?.label ?? ''}
                // totalStake={
                //   +formatBalance(resolvedIdentitiesSummary?.assets ?? '0', 18)
                // }  // Can't get TVL on created atoms at the moment
                variant={DataCreatedHeaderVariants.createdIdentities}
              >
                {atomsCreatedResult && (
                  <IdentitiesList
                    identities={atomsCreatedResult.atoms as Atom[]}
                    pagination={
                      atomsCreatedResult?.total?.aggregate?.count ?? 0
                    }
                    paramPrefix="createdIdentities"
                    enableSearch //
                    enableSort
                  />
                )}
              </TabContent>
            )}
          </Suspense>
```

We likely won't ever hit the loading state due to the Suspense fallback and we can probably streamline that, but this will show the Error boundary if there is an error in the query.

We can continue to optimize this pattern

```